web/gds-user-ui/src/modules/dashboard/member/utils.ts

```diff
@@ -3,11 +3,26 @@ import { convertToCVS, downloadCSV } from 'utils/utils';
 import { t } from '@lingui/macro';
 
 export const memberTableHeader = [
-  t`Member Name`,
-  t`Joined`,
-  t`Last Updated`,
-  t`Network`,
-  t`Status`
+  {
+    key: 'name',
+    label: t`Member Name`
+  },
+  {
+    key: 'joined',
+    label: t`Joined`
+  },
+  {
+    key: 'last_updated',
+    label: t`Last Updated`
+  },
+  {
+    key: 'network',
+    label: t`Network`
+  },
+  {
+    key: 'status',
```

Comment:
The BE returns a number for the verification state. In a separate PR, I plan on creating a util that will get the status based on what's provided from the BE. Here, we'll have to make sure that `VERIFIED` is displayed or users will be confused if they see a number. Here's a link to where the VerificationState is coming from: https://github.com/trisacrypto/trisa/blob/0b895ae982a8162b729f67e54b36e619ef39b8bf/pkg/trisa/gds/models/v1beta1/models.pb.go#L82C1-L82C1